duration.go

```diff
@@ -23,6 +23,18 @@ func NewDuration(reporter Reporter, value time.Duration) *Duration {
 	return newDuration(newChainWithDefaults("Duration()", reporter), &value)
 }
 
+// NewDurationC returns a new Duration instance with config.
+//
+// config should not be nil.
```

Comment:
Here and in other places.

> config should not be nil.

This comment doesn't make sense, config is not a pointer and you can't pass nil.

Instead, we have the following requirement:

> Either Reporter or AssertionHandler should not be nil.

However, instead of copy-pasting it, it's better to add a comment like this:

> Requirements for config are same as for WithConfig function.